core/server/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -79,6 +79,7 @@
   /**
    * This is only updated in the channel event loop thread when a read request starts or cancelled.
    * No need to be locked.
+   * When it is accessed outside of the event loop thread, make sure it is not null.
```

Comment:
can we be more clear here which methods here are called in event-loop? otherwise it is still not clear when we need to do the null check